specification/sql/resource-manager/Microsoft.Sql/preview/2021-11-01-preview/ManagedServerDnsAliases.json

```diff
@@ -337,6 +337,11 @@
           "description": "The fully qualified DNS record for managed server alias",
           "type": "string",
           "readOnly": true
+        },
+        "publicAzureDnsRecord": {
```

Comment:
do we need to update the example files?

Reply:
I regenerated the example files and they did not change, so we don't need to update them.

Reply:
Oh, I see what you mean, I will update them..

Reply:
Updated